helpers/fstab.py

```diff
@@ -13,15 +13,19 @@
 #
 # Open vStorage is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY of any kind.
+import inspect
+from ovs.extensions.generic.sshclient import SSHClient
+from ovs.extensions.generic.system import System
 
 
-class FstabHelper(file):
+class FstabHelper():
```

Comment:
When inheriting from nothing, it is implicitly inheriting from object (this changes in python 3.x though, thus better to write Object)